src/App.tsx

```diff
@@ -1,204 +1,60 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/header/header';
+import { Main } from './components/main/main';
+import { Footer } from './components/footer/footer';
+import { ErrorNotification } from './components/error/error';
+import { TodoFilter } from './types/filter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { handleFilter } from './utils/FilterFunction';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortFilter, setSortFilter] = useState<TodoFilter>(TodoFilter.All);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const areTodosActive =
+    todos.every(todo => todo.completed) && todos.length > 0;
+  const prepared = handleFilter(todos, sortFilter);
+  const activeCounter = todos.filter(todo => !todo.completed).length;
+  const notActiveCounter = todos.filter(todo => todo.completed).length;
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => setErrorMessage('Unable to load todos'));
+
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  }, []);
```

Comment:
It would be better to create a separate error handling function to avoid repetition in the future.
Smth like this: 

```suggestion
   const handleError = (error: string) => {
    setErrorMessage(error);

    setTimeout(() => {
      setErrorMessage('');
    }, 3000);
  };

  useEffect(() => {
    ...
      .catch(() => {
        handleError(ErrorMessage.LoadingError);
      ...
  }, []);

```